src/libs/ledger/ledger.ts

```diff
@@ -27,5 +27,8 @@ export const normalizeLedgerMessage = (error?: string): string => {
     return 'Rejected by your Ledger device.'
   }
 
-  return error
+  // Indicates a custom timeout error, no need to normalize
+  if (error.includes('Could not connect to your Ledger device for an extended period')) return error
+
+  return `Could not connect to your Ledger device. Please close any other apps that may be accessing your Ledger device (including wallet apps on your computer and web apps). Ensure your Ledger is responsive. Unplug and plug it again. Device error: ${error}`
```

Comment:
What if the device error is very long?

Reply:
Based on what I've seen, the device errors are relatively short, not descriptive and vague like "Invalid channel" and "Ledger Device is busy". I didn't manage to reproduce a lot longer error, so let's take the risk :v: